src/components/DatePicker/Day.jsx

```diff
@@ -2,13 +2,31 @@ import clsx from "clsx";
 import dayjs from "dayjs";
 import PropTypes from "prop-types";
 import React from "react";
+import { Dot } from "../Dot/Dot";
 
-export const Day = ({ selectedDate, date, getContent, currentMonth, disabled = false }) => {
+export const Day = ({ selectedDate, date, getContent, currentMonth, isLoading = false, disabled = false }) => {
     const isSameMonth = dayjs(currentMonth).isSame(date, "month");
+
     if (getContent && isSameMonth) {
+        if (isLoading) {
+            return (
+                <div className="mr-2 flex animate-pulse items-center justify-center">
+                    <Dot className="ui-day-content pointer-events-none !cursor-not-allowed" color="secondary" />
+                </div>
+            );
+        }
+
         return <DayContent selectedDate={selectedDate} date={date} getContent={getContent} />;
     }
 
+    if (isLoading) {
```

Comment:
What is the difference between this and the one above

Reply:
Firtst one displayed for Custom content and Dots there have bigger size, Second one displayed for Dates without custom content, only with the number of the month and they have fixed size